### PR TITLE
Update todoStorage.js

### PR DESCRIPTION
**Pull request by githubERIK**
I experimented with the code and Deployd backend (http://deployd.com/). It seemed to me that clearCompleted function worked better with my proposed code.

### CODE REVIEW

**Comment by samccone** [comment]
@githubERIK looks like a few style issues on CI would you mind fixing those up

**Comment by githubERIK** [comment]
Managed to fix those style errors.

**Comment by passy** [comment]
@githubERIK Could you explain why you want to change the code like this?

Also, it seems like you have found a loophole in our style guide, because this still looks very foreign compared to other code in that file and elsewhere. :)

**Comment by githubERIK** [comment]
With the master branch code I got "DELETE (my_localhost_URL_ path) 400 (Bad Request)" 
and the function did not delete records from Delpoyd and did not update the view. I changed the code and it seemed that it started to work. However, I just now noticed it will delete  only one record. Because of that, I now changed the code again and it seems to work well (_It will delete  all completed records_).
All in all, there is a chance that the original code in master branch did not work for me beacuse I might have modified my todoStorage.js http-paths the wrong way.

**Comment by addyosmani** [comment]
Travis still seems to be complaining here :monkey:

**Comment by githubERIK** [comment]
Indeed. Could somebody explain me, what Travis means by the following output?

```
...
/home/travis/build/tastejs/todomvc/tests/node_modules/drool/lib/index.js:27
  return (new webdriver.WebDriver.Logs(driver))
          ^
TypeError: webdriver.WebDriver.Logs is not a function
...
```

**Comment by githubERIK** [comment]
Pull request #1606 fixed Travis `webdriver.WebDriver.Logs` error. The Travis CI build passed.

File: examples/angularjs/js/services/todoStorage.js [hunk 1]
```diff
@@ -34,18 +34,12 @@ angular.module('todomvc')
...
-
-				angular.copy(incompleteTodos, store.todos);
-
-				return store.api.delete(function () {
-					}, function error() {
-						angular.copy(originalTodos, store.todos);
-					});
+				for (var i = 0; i < completeTodos.length; i++) {
+					store.delete(completeTodos[i]);
+				};
```
Comment (arthurvr) [inline examples/angularjs/js/services/todoStorage.js]: No need to have a semicolon here. Kinda surprises me that the linter doesn't check for this? :/

File: examples/angularjs/js/services/todoStorage.js [hunk 2]
```diff
@@ -34,18 +34,12 @@ angular.module('todomvc')
...
+					return todo.completed;
 				});
-
-				angular.copy(incompleteTodos, store.todos);
-
-				return store.api.delete(function () {
-					}, function error() {
-						angular.copy(originalTodos, store.todos);
-					});
+				for (var i = 0; i < completeTodos.length; i++) {
```
Comment (arthurvr) [inline examples/angularjs/js/services/todoStorage.js]: Just use `forEach`? :)

**Pull request closed without merging**